app/assets/stylesheets/style.scss

```diff
@@ -248,6 +248,28 @@ label:hover {
   }
 }
 
+.project-header {
+  text-align: center;
```

Comment:
Properties should be ordered background-color, border-radius, margin-bottom, margin-top, text-align